test/eredis_cluster_fakeredis_SUITE.erl

```diff
@@ -149,3 +151,105 @@ t_redis_crash(Config) when is_list(Config) ->
 %%     ?assertEqual({ok, undefined}, eredis_cluster:q(["GET","nonexists"])),
 
 %%     ?assertMatch(ok, eredis_cluster:stop()).
+
+t_ask_redirect(Config) when is_list(Config) ->
+    Ports = [20001, 20002],
+    fakeredis_cluster:start_link(Ports),
+    ok = eredis_cluster:connect([{"127.0.0.1", 20001}]),
```

Comment:
note: we should add ip6 capabilities to fakeredis_cluster so we can test ipv6 aswell